blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueRun.java

```diff
@@ -53,7 +56,7 @@
 
     /** Date String format */
     public static final String DATE_FORMAT_STRING = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
-
+    public static final DateTimeFormatter DATE_FORMAT = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
```

Comment:
Can we move joda specific code to AbstractRunImpl? This will also result in to `blue-rest` module not to depend on specific data time library.